.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/redux-async
+    docker:
+      - image: circleci/node:8
+    steps:
+      - checkout
+      - restore_cache:
+          key: npm-cache-v1-{{ checksum "package-lock.json" }}
+      - run:
+          name: Install Dependencies
+          command: npm ci
+      - save_cache:
+          key: npm-cache-v1-{{ checksum "package-lock.json" }}
+          paths:
+            - /home/circleci/.npm
+      - run:
```

Comment:
Yep, and add a second one for linting.

Reply:
<img width="294" alt="Screen Shot 2019-05-24 at 3 30 13 PM" src="https://user-images.githubusercontent.com/50588415/58352184-dc409880-7e38-11e9-9637-3a085c61c5b3.png">

is this the right thing to add?